backstop/tests/tag.html

```diff
@@ -390,13 +390,13 @@ <h2>Default tag containers</h2>
           tabindex="0"
```

Comment:
We need to change test background because `iui-visible` background is not so visible 😀

Reply:
Was this change done?

Reply:
Yes